javascript/index.js

```diff
@@ -23,18 +23,107 @@
   getInstruction("mashedPotatoes", 4, (step5) => {
     document.querySelector("#mashedPotatoes").innerHTML += `<li>${step5}</li>`;
     document.querySelector("#mashedPotatoesImg").removeAttribute("hidden");
-  }, (error) => console.log(error));
+  }, (error) => console.log(error)); */
 
 
 
 // Iteration 1 - using callbacks
-// ...
+getInstruction("mashedPotatoes", 0, (step1) => {
+  document.querySelector("#mashedPotatoes").innerHTML += `<li>${step1}</li>`;
+
+  getInstruction("mashedPotatoes", 1, (step2) => {
+    document.querySelector("#mashedPotatoes").innerHTML += `<li>${step2}</li>`;
+
+    getInstruction("mashedPotatoes", 2, (step3) => {
+      document.querySelector("#mashedPotatoes").innerHTML += `<li>${step3}</li>`;
+
+      getInstruction("mashedPotatoes", 3, (step4) => {
+        document.querySelector("#mashedPotatoes").innerHTML += `<li>${step4}</li>`;
+
+        getInstruction("mashedPotatoes", 4, (step5) => {
+          document.querySelector("#mashedPotatoes").innerHTML += `<li>${step5}</li>`;
+          document.querySelector("#mashedPotatoesImg").removeAttribute("hidden");
+
+          document.querySelector("#mashedPotatoes").innerHTML += `<li>${'Mashed Potates are ready!'}</li>`;
+
+        }, (error) => console.log(error)); 
+
+      }, (error) => console.log(error));
+
+    }, (error) => console.log(error));
+
+  }, (error) => console.log(error));
+  
+}, (error) => console.log(error));
 
 // Iteration 2 - using promises
```

Comment:
Here instead of chaining the .then inside of each other, you should have separate .then()